server/utils/auth.js

```diff
@@ -21,7 +22,7 @@ module.exports = {
     }
 
     try {
-      const { data } = jwt.verify(token, secret, { maxAge: expiration });
+      const { data } = jwt.verify(token, JWT_SECRET, { maxAge: expiration });
       req.user = data;
```

Comment:
Leave this part how you had it.  Just change the value of the secret variable to process.env.JWT_SECRET